connection.go

```diff
@@ -991,6 +992,11 @@ func (c *Connection) StatusBoard() *statusboard.Client {
 	return statusboard.NewClient(c, "/api/status-board")
 }
 
+// ServiceMgmt returns the client for the service management service.
+func (c *Connection) ServicMgmt() *servicemgmt.Client {
+	return servicemgmt.NewClient(c, "/api/service_mgmt")
+}
+
```

Comment:
@renan-campos when a new service is added most of the code is generated automatically, but this tiny method has to be written manually.